htdocs/core/tpl/massactions_pre.tpl.php

```diff
@@ -43,6 +43,7 @@
 		$listofselectedref = array();
 		foreach ($arrayofselected as $toselectid)
 		{
+			$objecttmp = new $objectclass( $db );
```

Comment:
Mass actions are designed to work on high number of selected elements. If we init a new object here for each element select, we create an array that may consume a very high amount of memory with a high risk to create memory lacks or server overlapping. Is this really required ?